tests/expressions/tests.py

```diff
@@ -1043,6 +1043,38 @@ def test_duration_with_datetime(self):
         ).order_by('name')
         self.assertQuerysetEqual(over_estimate, ['e3', 'e4'], lambda e: e.name)
 
+    @skipUnlessDBFeature('supports_microsecond_precision')
+    def test_negative_timedelta_with_microsecond_support_update(self):
+        # subtract 30 microseconds, 30 seconds, 30 minutes, 2 hours and 2 days
+        Experiment.objects.filter(name='e0').annotate(
```

Comment:
@felixxm I think what Tim means is using different aliases for each annotations.